src/components/common/Certify.vue

```diff
@@ -51,7 +51,7 @@
           <strong>Date:</strong> {{ formattedCurrentDate || '[unknown]' }}
         </p>
         <p class="certify-clause">
-          {{ message }}
+          <strong v-if="message">Note:</strong> {{ message }}
         </p>
```

Comment:
Updated the format per design, so needed to update the CertifyText for all business types